### DISCUSSION TITLE
Does version 6 really require Go 1.21, or can I use 1.19?

### DISCUSSION THREAD

**Discussion by davidflanagan** [santhosh-tekuri/jsonschema#186] [Q&A] [answered]
I'm stuck using go 1.19 and wondering if I'll be able to use version 6. go.mod says 1.21, but I'm wondering if there are actually any dependencies on that version. It looks like I can use version 5.3, but I'd rather use the latest if it is possible...

**Comment by santhosh-tekuri** [comment]
I am not sure. Better checkout repo and build it with go 1.19 to find out.

**Comment by davidflanagan** [comment, marked as answer]
v6 appears to use the slices package, which was new in 1.21, I think. So it does not work with 1.19.
